src/git.md

```diff
@@ -44,14 +44,19 @@ git remote add upstream git@github.com:rust-lang/rust.git
 
 if you're using SSH.
 
+**note:** This page is dedicated to workflows for `rust-lang/rust`, but will likely be
```

Comment:
We usually use capital "Note":

```suggestion
**Note:** This page is dedicated to workflows for `rust-lang/rust`, but will likely be
```

Reply:
actually rest of file does NOTE so ill use that